Sources/Utility/Git.swift

```diff
@@ -57,7 +57,6 @@ public class Git {
         return knownVersions
     }
 
-    public class var tool: String {
-        return "git"
-    }
+    /// A shell command to run for Git. Can be either a name or a path.
+    public static var tool: String = "git"
```

Comment:
The intent here is to be able to customize the first argument of the command line array in the same way as one can do it the shell:  it can be either a string, which will be found using PATH, or it can be a path (relative or absolute), in which case PATH won't be consulted.  We don't have a good way to represent that right now — a path that could be a string, a relative path, or an absolute path.  Maybe we need a ShellPathSearchableNameOrPath protocol.   :)